.travis.yml

```diff
@@ -3,5 +3,12 @@ language: node_js
 node_js: 8
 script:
   - npm run-script build
+  - docker build -t dredd-data-broker . -f ./tests/Dockerfile
+  - cd tests
+  - docker-compose run data-broker
+before_script:
+  - cd tests
```

Comment:
I'd avoid stepping into directories on travis (this has been the root of way too many issues up to now)

perhaps?
```yaml
script:
  - npm run-script build
  - docker build -t dredd-data-broker . -f tests/Dockerfile
  - docker-compose -f tests/docker-compose.yaml run data-broker
before_script:
  - docker-compose -f tests/docker-compose.yaml up -d kafka data-broker-redis
```

Reply:
I'll remove the cd's. I did the same for auth.